### DISCUSSION TITLE
Issue: Doesn't download videos from TheWorldWatch.com

### DISCUSSION THREAD

**Discussion by alkapwn3d** [aclap-dev/video-downloadhelper#174] [Help and Q&A]
### Your question

Since theres no Issues tab I'll just say it here

site: theworldwatch.com
browser: firefox

addon doesnt recognize what videos to download anymore. I can still download the video by getting the link from the dev-tools networks tab.

### Addon & CoApp version and details

addon: video download helper
browser: firefox
os: linux

**Comment by Mattmjs** [comment]
No problem on that site using:  Version 8.2.0.23 (stable) + VdhCoApp 2.0.19 on windows 10.

They're simple mp4 files, so not sure why you don't have detection of media.

**Reply by alkapwn3d** [reply]
does it download the file in firefox when you click any of them?
doesnt for me

**Comment by paulrouget** [comment]
@alkapwn3d we need more info that that: https://github.com/aclap-dev/video-downloadhelper/wiki/Where-can-I-find-my-versions%3F

**Comment by alkapwn3d** [comment]
let me know what other info you need

version: 8.2.0.23

i see the correct file in the context menu but it doesnt start the download

**Comment by paulrouget** [comment]
> let me know what other info you need

Everything I need is explained in the link I shared.
